control.rb

```diff
@@ -111,27 +115,39 @@ def make_actions
   end
 
   def player_action
-    border
-    action_selection
-    action = gets.chomp.to_i
+    @interface.borderline
+    @interface.show_action_selection_menu
+    action = @interface.get_answer.to_i
     case action
     when 1
-      skip_a_move
+      @interface.player_skip_the_action_message
     when 2
-      player_take_a_card
+      @interface.player_take_a_card_message
       add_card(@player)
       show_user_his_hand
       show_user_points
+      check_overflow
     when 3
       show_cards
     else
-      wrong_input
+      @interface.wrong_input_message
     end
-    border
+    @interface.borderline
   end
 
-  def skip_a_move
-    return
+  def overflow?(participant)
+    participant.hand.hand_worth > MAX_POINTS_TO_WIN ? true : false
```

Comment:
```suggestion
    participant.hand.hand_worth > MAX_POINTS_TO_WIN
```
сравнение само по себе возвращает булево